utils/airsim_manager.py

```diff
@@ -133,25 +128,17 @@ def get_cars_distance(self):
             np.array([[car2_position_and_speed["x"], car2_position_and_speed["y"]]])))
         return dist_c1_c2
 
-    def get_car1_state(self, logger):
+    def get_car1_state(self, logger=None):
```

Comment:
car state should be as defined in notion, am I missing something? :)

Reply:
fixed